proxy.go

```diff
@@ -113,17 +113,25 @@ func (p *Proxy) SetMITM(config *mitm.Config) {
 	p.mitm = config
 }
 
-// Close sets the proxying to the closing state and waits for all connections
-// to resolve.
+// Close sets the proxy to the closing state, finishes inflight requests, and actively closes all connections.
```

Comment:
instead of `and actively closes all connections.`
`and waits for all connections to close or time out`.